src/Components/Footer.js

```diff
@@ -15,7 +15,7 @@ function Footer() {
         <a href="https://github.com/EddieHubCommunity/LinkFree">
           <i className="pi pi-github" aria-hidden="true"></i>
         </a>
-        v{version}
+        &nbsp;v{version}
```

Comment:
This is probably better done with styling.

I think the version number could be wrapped in a `span` then have styling added to it